compose/wordpress.md

```diff
@@ -40,7 +40,7 @@ Compose to set up and run WordPress. Before starting, you'll need to have
        db:
          image: mysql:5.7
          volumes:
-           - db_data:/var/lib/mysql
+           - ./db_data:/var/lib/mysql
```

Comment:
This is incorrect. If the first field is a name rather than a path, this indicates that it is a **volume**, rather than a **bind mount**. See https://docs.docker.com/engine/admin/volumes/.

Reply:
You are right @mstanleyjones, sorry for the bad PR